lmnet/tests/lmnet_tests/datasets_tests/test_image_folder.py

```diff
@@ -75,7 +75,7 @@ def test_image_folder():
 
 class DummyHasValidation(ImageFolderBase):
     extend_dir = "dummy_classification"
-    validation_extend_dir = "lm_things_on_a_table"
+    validation_extend_dir = "open_images_v4"
```

Comment:
`OpenImageV4` class doesn't derived from `ImageFolderBase` class.  I'm not sure whether this change is OK.

Reply:
yeah. That makes sense. both `lm_things_on_a_table` and `open_images_v4` is not related to `ImageFolderBase` AFAIK. Maybe it's better to extend to `dummy_classification` as well right? Since there is no `ImageFolder` dataset fixture.

Reply:
I check the whole implementation of this file. Current implementation just checks some property of dataset (say, `num_per_epoch`, `extend_dir`, etc) but doesn't check that whether `__getitem__` method work, which is the most important feature for dataset class. It looks better to add some new unit tests, could you add a new github issue?

For this pull request, let's leave this as-is.

Reply:
@tkng Issue created here:
https://github.com/blue-oil/blueoil/issues/509

Reply:
Thanks, other than that, LGTM.